Add SQL reference content batch #7 (#2672)
fishtown-analytics/docs.getdbt.com: website/docs/sql-reference/operators/sql-not.md

---
id: not
title: SQL NOT
description: The SQL NOT operator allows you to return results from conditions that are not true. he NOT boolean is kind of similar to an adjective—it’s often put in front of another operator. 
slug: /sql-reference/not
---

<head>
    <title>Working with the SQL NOT operator</title>
</head>

This will be a not *not* useful page on a helpful SQL operator.

Ok we had to get that out of the way. The SQL NOT operator allows you to return results from conditions that are not true. Pretty intuitive, right?

In this page, we’ll dive into how to use the NOT operator, demonstrate an example, and elaborate on potential use cases.

## How to use the SQL NOT operator

> The NOT boolean is kind of similar to an adjective—it’s often put in front of another operator, such as BETWEEN, LIKE/ILIKE, IS, and IN, to return rows that do not meet the specified criteria. Below is an example of how to use NOT in front of a LIKE operator:

Reviewer: ```suggestion
The NOT boolean is kind of similar to an adjective—it’s often put in front of another operator, such as [BETWEEN](/sql-reference/between), [LIKE](sql-reference/like)/[ILIKE](/sql-reference/ilike), IS, and [IN](/sql-reference/in), to return rows that do not meet the specified criteria. Below is an example of how to use NOT in front of a LIKE operator:
```

> The NOT boolean is kind of similar to an adjective—it’s often put in front of another operator, such as [BETWEEN](/sql-reference/between), [LIKE](sql-reference/like)/[ILIKE](/sql-reference/ilike), IS, and [IN](/sql-reference/in), to return rows that do not meet the specified criteria. Below is an example of how to use NOT in front of a LIKE operator:

Reviewer: ```suggestion
The NOT boolean is kind of similar to an adjective—it’s often put in front of another operator, such as [BETWEEN](/sql-reference/between), [LIKE](/sql-reference/like)/[ILIKE](/sql-reference/ilike), IS, and [IN](/sql-reference/in), to return rows that do not meet the specified criteria. Below is an example of how to use NOT in front of a LIKE operator:
```

`where <field_name> not like <value>`

This syntax can be easily modified for other operators:
- `where not between <value_1> and <value_2>`
- `where <field_name> is not null`
- `where <field_name> is not in (array_of_options)`

> - …or placed altogether in a different place, such as a case statement (ex. `case when <field_name> is not null then1 else 0 end`)

Reviewer: ```suggestion
- …or placed altogether in a different place, such as a case statement (ex. `case when <field_name> is not null then 1 else 0 end`)
```
